src/fluree/util.clj

```diff
@@ -112,8 +112,10 @@
                                       :else
                                       (try (Integer/parseInt v) ;; parse to integer if possible
                                            (catch Exception _ v)))]
-                             (assoc acc k* v*))) {} opts)]
-    [opts-map args*]))
+                             (assoc acc k* v*))) {} opts)
+        ;; parse any args possible into integers
+        args**   (map #(try (Integer/parseInt %) (catch Exception _ %)) args*)]
+    [opts-map args**]))
```

Comment:
Do we want to try parsing all args as ints and keep any that parse (if I'm reading it correctly)? This seems like a great application of clojure.spec and input coercion.

Reply:
There are only 4 possible args, and the only time an int will be used is to specify blocks... so this approach has no functional downside, no performance downside... only a visual downside. I'd rather write a function that used a regex and then more precisely parsed to integer (which is what parseInt does) than pull in a library dependency for just this purpose. I'm not sure I like the idea of spec being used for argument coercion (not sure I don't like it either, but that's not the design intention of clojure.spec so far as I know)... but for use in testing I'm a big fan.